baywatch.install

```diff
@@ -324,3 +324,33 @@ function baywatch_update_8013() {
   $baywatch = new BaywatchOperation();
   $baywatch->enable_tide_ui_restriction();
 }
+
+/**
+ * Remove Elasticsearch Connector configuration from config_split
+ */
+function baywatch_update_8014() {
+    // config_split files
+    $configs = [
+        'config_split.config_split.ci' => 'config_split',
+        'config_split.config_split.dev' => 'config_split',
+        'config_split.config_split.local' => 'config_split',
+    ];
+    /*
+     * IF elasticsearch_connector.cluster.elasticsearch_bay in greylist
+     * THEN remove it from the list
+     */
+    foreach ($configs as $config => $type) {
+        $config_file = DRUPAL_ROOT . '/' . drupal_get_path('module', 'baywatch') . '/config/optional/' . $config . '.yml';
+        $inactive_config_values = Yaml::decode(file_get_contents($config_file));
+        $inactive_graylist = $inactive_config_values['graylist'];
```

Comment:
line 343 to 345 is not required. Do not need to check the inactive settings.

Reply:
Done